cclib/parser/molcasparser.py

```diff
@@ -237,6 +243,14 @@ def extract(self, inputfile, line):
                 target = float(line.split()[-1])
                 scftargets.append(target)
 
+            self.skip_lines(inputfile,['Threshold for linear dependence', 'Threshold at which DIIS is turned on', \
+                                         'Threshold at which QNR/C2DIIS is turned on'])
+            line = next(inputfile)
+
+            if line[6:31] == 'Threshold for Norm(delta)':
+                target = float(line.split()[-1])
+                scftargets.append(target)
+
```

Comment:
@berquist @langner @ATenderholt This section is included because after properly reading the Molcas manual I got to know that there was another criteria for convergence of SCF Energy. So I included it in `scftargets` and `scfvalues`.

Reply:
nit: I would prefer the SCF stuff to be done in a separate change

If left here, please update the PR description to reflect this.

Reply:
@langner I will make them in separate PR.